utils/rpms/daos.spec

```diff
@@ -539,6 +539,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Sat Apr  2 2022 Joseph Moore <joseph.moore@intel.com> 2.1.100-25
+- Update to mercury 2.0.1.rc4-8 to include UCX provider patch
```

Comment:
looks like we have typo here and in previous comments too. should be mercury 2.1.0rc4-8 not 2.0.1rc4-8

Reply:
Please address